### PR TITLE
PR for activity Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by DarioAlessioR**
PR for activity Portfolio: mobile menu

This activity is part of week 4 of Module 1 of Microverse program, according to the following instructions:

In mobile, implement the following features:

When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Thank you in advance for your kind review.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @DarioAlessioR, @JoffreyNK,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not passed the linters check

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -233,5 +236,11 @@ <h3>emailhere@email.com</h3>
     
     </footer>
 
+    <script src="./index.js"> </script>
+
+
+
+
+
```
Comment (zieeco) [inline index.html]: - [ ] **[OPTIONAL]**

- Please, kindly remove these added now lines as it adds too many new lines in your HTML file which is not ideal and makes your codebase not clean.
- Your portfolio is having a horizontal scroll. please adjust your styling to fix that to improve the user experience

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Team


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
_check the comments under the review_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DarioAlessioR) [reply index.html]: Thank you for your review and optional suggestions @zieeco

**Comment by DarioAlessioR** [comment]
Merge menu1 to master after approval of project

**Pull request merged**